stdlib/2.7/xml/sax/__init__.pyi

```diff
@@ -0,0 +1,20 @@
+# Stubs for xml.sax (Python 2.7)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+class SAXException(Exception):
+    def __init__(self, msg, exception=None): ...
```

Comment:
Oh wait, can't you manually add types to these signatures, insofar as these methods are documented?